fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -751,24 +755,95 @@ def set_field(tokenizers_info_cfg: List, name: str, key: str, val: Any) -> List:
         with open(os.path.join(path, "config.yaml"), "w") as f:
             OmegaConf.save(tokenizer_config_overall, f)
 
-    def _add_single_tokenizer(
+    def update_special_tokens(self, added_tokens: List, save_tokenizer_path: str = None):
```

Comment:
Are you sure there's a point in adding this function to the class?

Reply:
Actually, this was done intentionally.  The function modifies the given modular tokenizer internally, so it should be inside the class.  Buy the real reason is that the old version would modify the tokenizer and then return it, and this behavior seems to hint that the returned tokenizer is a different one, as was evident from the usage code.  This way, it is clearer that the function operates on the object and modifies it.
And finally, it's used in two different scripts now, and importing between scripts does not feel like a good practice.

Reply:
Agree